cba/docs/sample-project/1-configure-content-libraries.md

```diff
@@ -37,7 +37,7 @@ Create a content library for storing the images that you can then use to deploy
 1. Navigate to the Terraform example in the repository.
 
    ```bash
-   cd terraform-sample-project/01-vsphere-content-library-publishing
+   cd /validated-solutions-for-cloud-foundation/cba/terraform-sample-project/01-vsphere-content-library-publishing
```

Comment:
I think this first `/` in the path should be removed here and in the other `.md` files.

Reply:
Related comment. It just occurred to me that that samples use `/` for Linux (`bash`) versus `\` for Windows.

Reply:
First / is needed, the Implementation page I Updated so you can copy / paste in the way you had it before you have to do cd ..\.. before you can then do the actual cd etc